tests/test_annotate_anndata.py

```diff
@@ -0,0 +1,92 @@
+import numpy as np
+import pandas as pd
+import pytest
+from anndata import AnnData
+
+import genomic_features as gf
+
+genes = gf.ensembl.annotation("Hsapiens", 108).genes()
+
+# Make dummy anndata object
+adata = AnnData(
+    X=np.array([[1, 2, 3], [4, 5, 6]]),
+    obs=pd.DataFrame(index=["cell1", "cell2"]),
+    var=pd.DataFrame(index=["ENSG00000278862", "ENSG00000212694", "ENSG00000113721"]),
+)
```

Comment:
Could these go in a fixture or something? To avoid mutating state between tests.